examples/apps/quick_tap.py

```diff
@@ -0,0 +1,415 @@
+        self.cube.stop_light_chaser()
+
+    async def determine_move(self, buzzer_display_type):
+        '''Cozmo chooses a move based on the probabilities above.
+
+        Args:
+            buzzer_display_time (string): the display of the buzzers, either RED, SAME, or DIFFERENT
+        '''
+        await self.hesitate()
+        probability_correct = random.random()
+        if probability_correct < RATE_COZMO_ACCURACY:
+            if buzzer_display_type == BUZZER_SAME:
+                await self.tap()
+            else:
+                await self.not_tap()
+        else:
+            if buzzer_display_type == BUZZER_RED or buzzer_display_type == BUZZER_DIFFERENT:
+                await self.tap()
+            else:
+                await self.not_tap()
+
+    async def hesitate(self):
+        '''Cozmo waits between 0 and 0.5 seconds'''
+        delay = random.random() * .5
+        await asyncio.sleep(delay)
+
+    async def tap(self):
+        '''Calls Cozmo's tap animation.'''
+        await self.robot.play_anim_trigger(cozmo.anim.Triggers.OnSpeedtapTap).wait_for_completed()
+
+    async def not_tap(self):
+        '''Randomly calls either Cozmo's fakeout tap animation or his idle animation.'''
+        probability_fakeout = random.random()
+        if probability_fakeout < 0.5:
+            await self.robot.play_anim_trigger(cozmo.anim.Triggers.OnSpeedtapFakeout).wait_for_completed()
+        else:
+            await self.robot.play_anim_trigger(cozmo.anim.Triggers.OnSpeedtapIdle).wait_for_completed()
+
+
+class BlinkyCube(cozmo.objects.LightCube):
+    ''' Same as a normal cube, plus method to display moving rotating rainbow lights.
+
+        Args:
+            same arguments as cozmo.objects.LightCube
+    '''
+    def __init__(self, *a, **kw):
+        super().__init__(*a, **kw)
+        self._chaser = None
+
+    def start_light_chaser(self):
+        '''Rotates four colors around the cube light corners in a continuous loop.'''
+        if self._chaser:
+            raise ValueError("Light chaser already running")
+        async def _chaser():
+            rainbow_colors = [cozmo.lights.blue_light, cozmo.lights.red_light, cozmo.lights.green_light, cozmo.lights.white_light]
+            while True:
+                for i in range(4):
+                    self.set_light_corners(*rainbow_colors)
+                    await asyncio.sleep(0.1, loop = self._loop)
+                    light = rainbow_colors.pop(0)
+                    rainbow_colors.append(light)
+        self._chaser = asyncio.ensure_future(_chaser(), loop = self._loop)
+
+    def stop_light_chaser(self):
+        '''Ends the _chaser loop.'''
+        if self._chaser:
+            self._chaser.cancel()
+            self._chaser = None
+        self.set_lights_off()
+
+    async def countdown(self):
+        '''Sets all lights to white, then 3, then 2, then 1, then 0.'''
```

Comment:
"then 3, then 2, then 1, then 0" - This is referring to how many lights are white? Maybe we could describe this differently as it wasn't clear to me. Maybe: `Sets all lights to white, then 3 lights, 2 lights, 1 light, then none.`